samples/timestamp.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 The Amber Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "samples/timestamp.h"
+
+#include <cassert>
+
+#include "amber/platform.h"
+
+#if AMBER_PLATFORM_WINDOWS
+// TODO: windows include?
+#elif AMBER_PLATFORM_POSIX
+#include <time.h>
+#endif
+
+namespace timestamp {
+
+long SampleGetTimestamp() {
+  long timestamp = 0L;
+#if AMBER_PLATFORM_WINDOWS
+  assert(false && "TODO: implement GetTimestamp() on Windows");
```

Comment:
https://pdfium.googlesource.com/pdfium/+/refs/heads/master/core/fxcrt/fx_random.cpp#55  is what you want I think. Looks like there is no include required, just use GetSystemTime here

Reply:
I think we might prefer QueryPerformanceCounter (see https://docs.microsoft.com/en-us/windows/desktop/SysInfo/acquiring-high-resolution-time-stamps) , as GetSystemTime precision is milliseconds at best, and it is affected by change of system time (e.g. summer time, etc).